src/testing/operate.test.js

```diff
@@ -0,0 +1,20 @@
+import operate from '../logic/operate';
+
+describe('Operate calculation', () => {
+  test('9 plus 9', () => {
+    const result = operate(9, 9, '+');
+    expect(result).toBe('18');
+  });
+  test('9 minus 9', () => {
+    const result = operate(9, 9, '-');
+    expect(result).toBe('0');
+  });
+  test('9 times 9', () => {
+    const result = operate(9, 9, 'x');
+    expect(result).toBe('81');
+  });
+  test('9 divided 9', () => {
+    const result = operate(9, 9, '/');
+    expect(result).toBe('1');
+  });
+});
```

Comment:
- [ ] Good job so far 👏 . Kindly add the modulus test ti make sure all edge cases are taken care of 👍 

### View:
![image](https://github.com/GhausAliKhan/math-magicians/assets/50990925/673c98fa-9fc9-4a10-b8a3-77d375d356dd)